configAnchorMsg:
  nAnchors: 3
  anchors: [
    [0.0, 0.0, 0.0],
    [10.0, 10.0, 10.0],
    [20.0, 20.0, 20.0],
    [50.0, 50.0, 10.0],
    [50.0, 0.0, 10.0]
  ]